Code/Tools/FBuild/FBuildCore/Protocol/Protocol.h

```diff
@@ -31,7 +31,7 @@ namespace Protocol
     enum : uint16_t { PROTOCOL_PORT = 31264 }; // Arbitrarily chosen port
 
     // Protocol Version
-    enum : uint32_t { PROTOCOL_VERSION_MAJOR = 22 };    // Changes here make workers incompatible
+    enum : uint32_t { PROTOCOL_VERSION_MAJOR = 22 + 1 };    // Changes here make workers incompatible
```

Comment:
This will force all users to update all their workers, which is burdensome for some users with large build farms and multiple dev teams operating on different versions.

A change like this needs to be made in a backwards compatible. way.